pyroute2/dhcp/dhcp4msg.py

```diff
@@ -40,6 +40,7 @@ class dhcp4msg(dhcpmsg):
         (Option.COOKIE_SERVER, 'ip4list'),
         (Option.LPR_SERVER, 'ip4list'),
         (Option.HOST_NAME, 'string'),
+        (Option.MTU_SIZE, 'be16'),
```

Comment:
I think simply `MTU` would be enough since it's already a size in itself. Or maybe `INTERFACE_MTU` since it's the name in RFC 2132.